src/App.jsx

```diff
@@ -14,45 +16,142 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const ALPHABET = 'Alphabet';
+const LENGTH = 'Length';
+const REVERSE = 'Reverse';
+
+function getPreparedGoods(goods, sortFiled, reverse) {
+  const preparedGoods = [...goods];
+
+  if (sortFiled) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortFiled) {
+        case ALPHABET:
+          return good1.localeCompare(good2);
+
+        case LENGTH:
+          return good1.length - good2.length;
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (reverse === REVERSE) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
+export const App = () => {
+  // const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
+  const [sortFiled, setSortFiled] = useState('');
+  const [reverse, setReverse] = useState('');
```

Comment:
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```